src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+2.7.25
```

Comment:
Should be removed, bug fixes are documented in the main CHANGELOG, which is automatically generated when I make a new release.